deploy/src/config.rs

```diff
@@ -0,0 +1,20 @@
+use clap::Parser;
+
+/// Deploy operator config
+#[derive(Debug, Parser)]
+#[non_exhaustive]
+#[command(author, version, about, long_about = None)]
+pub struct Config {
+    /// The address on which the HTTP server will listen to
+    #[arg(long, default_value = "0.0.0.0:8080")]
+    pub listen_addr: String,
+    /// The namespace to deploy
+    #[arg(long, default_value = "default")]
+    pub namespace: String,
+    /// Enable operator to work in all namespaces
```

Comment:
How many namespaces do we have? In what namespaces do we need to enable the operator to work and in what not?

Reply:
These codes are introduced in #1.
The number of namespaces depends on the situation of k8s, and this parameter is determined by the user. By the way, there is a `cluster_wide` parameter below, which will allow deploy-operator to work in all namespaces.